test-data/unit/fixtures/args.pyi

```diff
@@ -27,3 +27,4 @@ class int:
 class str: pass
 class bool: pass
 class function: pass
+class ellipsis: pass
```

Comment:
This was needed because a few tests that import collections didn't have `...` available in their fixtures, and my stub for namedtuple uses it.